.github/workflows/e2e-test.yml

```diff
@@ -32,7 +39,7 @@ jobs:
         echo "export const config = $PROJECT_CONFIG; export const testAccount = $TEST_ACCOUNT" > firebase-config.js
     - name: Yarn install
       run: |
-        yarn add firebase
+        yarn add firebase@next
```

Comment:
Instead of hardcoding it to the `next` tag, can the version be passed in from the caller?

Reply:
Ok, this now takes the tag from the input (defaults to 'next'). I have a parallel change in release-cli ready to send the version and tag inputs, which I can merge when this is merged.